dms/tools/file.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 Antoni Romera
+# Copyright 2017-2019 MuK IT GmbH
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import mimetypes
+import os
+import re
+import shutil
+import tempfile
+import unicodedata
+
+from odoo.tools.mimetypes import guess_mimetype
+
+
+def slugify(value, lower=True):
```

Comment:
This tool already exists in Odoo. It should use that one IMO.

Reply:
Removed